.github/workflows/test.yml

```diff
@@ -146,46 +159,6 @@ jobs:
                 --override-input haskell-backend . --update-input haskell-backend \
                 --command bash -c "cd test && make -j2 --output-sync test"
 
-  cabal:
-    name: 'Cabal / Unit Tests'
```

Comment:
No `cabal` build any more?

Reply:
the nix job calls cabal directly so no point in having a separate one